packages/@ember/-internals/glimmer/tests/integration/application/engine-test.js

```diff
@@ -273,7 +273,7 @@ moduleFor(
         `The use of \`{{partial}}\` is deprecated, please refactor the "troll" partial to a component`
       );
       expectDeprecation(
-        'Using {{attrs}} to reference named arguments has been deprecated. {{attrs.wat}} should be updated to {{@wat}}. (L1:C2) '
+        'Using {{this.attrs}} to reference named arguments has been deprecated. {{this.attrs.wat}} should be updated to {{@wat}}. (L1:C2) '
```

Comment:
Something is off target- we should not still have a deprecation. `{{this.attrs}}` should work in an Ember component context because it behaves like any other property access, and `attrs` are set on Ember components.

The goal of this removal is to: a) drop any special transform of `{{attrs}}` b) removed the deprecation warning. That likely has the side effect that `{{this.attrs}}` is no longer deprecated, and that is ok.

Reply:
Yup, this was a mistake on my part. Odd that it didn't fail even though there was no deprecation.

Removed